scwx-qt/source/scwx/qt/manager/marker_manager.cpp

```diff
@@ -198,17 +239,41 @@ MarkerManager::Impl::GetMarkerByName(const std::string& name)
 
 MarkerManager::MarkerManager() : p(std::make_unique<Impl>(this))
 {
+   const std::vector<types::MarkerIconInfo> defaultMarkerIcons_ {
```

Comment:
This could be made static, and then used by reference in the thread pool below to avoid a copy. Or, defined inside the lambda.